src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -153,4 +159,13 @@ public String getPath() {
         return path.toString();
     }
 
+    /**
+     * Show warning message to user if needed.
+     * 
+     * @param message to be shown to user
+     */
+    private void showWarningMessage(String message) {
+        TextUi textUi = new TextUi();
+        textUi.showToUser("", message, "");
+    }
```

Comment:
So I'm supposed to throw exception to stop the program? o.o
Or just catch and let it pass while keeping this warning message?